rest-api/src/test/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/CaseDefinitionControllerTest.java

```diff
@@ -35,12 +36,13 @@
 
     private CaseTypeService caseTypeService = mock(CaseTypeService.class);
     private JurisdictionService jurisdictionService = mock(JurisdictionService.class);
+    private CaseRoleService caseRoleService = mock(CaseRoleService.class);
     private CaseDefinitionController subject;
     private MockMvc mockMvc;
 
     @BeforeEach
     public void createSubject(){
-       subject = new CaseDefinitionController( caseTypeService, jurisdictionService );
+       subject = new CaseDefinitionController( caseTypeService, jurisdictionService, caseRoleService);
        mockMvc = MockMvcBuilders.standaloneSetup(subject)
```

Comment:
space after bracket '(' Static analysis tool does not like it

Reply:
👍